src/Grids/vertical_discretization.jl

```diff
@@ -68,7 +69,7 @@ or an `AbstractArray`. A `MutableVerticalDiscretization` defines a vertical coor
 following certain rules. Examples of `MutableVerticalDiscretization`s are free-surface following coordinates,
 or sigma coordinates.
 """
-MutableVerticalDiscretization(r) = MutableVerticalDiscretization(r, r, (nothing for i in 1:9)...)
+MutableVerticalDiscretization(r) = MutableVerticalDiscretization(r, r, (nothing for i in 1:10)...)
```

Comment:
this give `r` both for face and for centers, right? how does this work?

Reply:
This is just a constructor for the user interface. The coordinate is populated when constructing the grid (this vertical coordinate also needs horizontal information).
So that users can do
```julia
z = MutableVerticalDiscretization(r_faces)
grid = RectilinearGrid(; z, size= ... , )
```